Dockerfile

```diff
@@ -1,19 +1,20 @@
-FROM ubuntu:disco
+FROM ubuntu:focal
 
 ARG MAKEFLAGS=-j2
 
 ENV GOROOT /opt/go
 ENV PATH ${GOROOT}/bin:${GOPATH}/bin:${PATH}
 ENV NFF_GO /nff-go
 
+RUN DEBIAN_FRONTEND="noninteractive" apt-get -q update && apt-get -y install tzdata
```

Comment:
tzdata gets installed and hangs for user input on the timezone. This allows to install the package with a default timezone.